boards/common/blxxxpill/include/periph_conf.h

```diff
@@ -174,23 +195,35 @@ static const uart_conf_t uart_config[] = {
         .rx_pin     = GPIO_PIN(PORT_A, 10),
         .tx_pin     = GPIO_PIN(PORT_A, 9),
         .bus        = APB2,
-        .irqn       = USART1_IRQn
+        .irqn       = USART1_IRQn,
+#ifdef MODULE_PERIPH_DMA
+        .dma        = 2,
+        .dma_chan   = 2
```

Comment:
The `.dma_chan` is not used on the stm32f1 (and similar). Can you amend your commit and set all `.dma_chan` members to zero and maybe put a comment after each mentioning that the value is unused. Something like:
```suggestion
        .dma_chan   = 0 /* Unused on the stm32f1 */
```